perfkitbenchmarker/sample.py

```diff
@@ -57,11 +58,18 @@ class Sample(collections.namedtuple('Sample', _SAMPLE_FIELDS)):
     value: float. Result for 'metric'.
     unit: string. Units for 'value'.
     metadata: dict. Additional metadata to include with the sample.
+    timestamp: float. Unix timestamp.
   """
 
-  def __new__(cls, metric, value, unit, metadata=None, **kwargs):
+  def __new__(cls, metric, value, unit, metadata=None, timestamp=None,
+              **kwargs):
+    if timestamp is None:
+      timestamp = time.time()
```

Comment:
I sent out emails more than a day ago and haven't heard anyone object. I think it's safe to make the change.